commands/member.js

```diff
@@ -31,13 +32,14 @@ exports.role = function(message, args){
 
 exports.rmrole = function(message, args){
     let role = message.guild.roles.find(`name`, args.join(' '));
+    logger.info(`User ${message.author.username} leaving role ${role}`);
     if (role){
-    message.member.removeRole(role).catch(console.error);
-    message.channel.send(`Yor are no longer a member of: ${args.join(' ') }... \nSorry to see you go` );
-    return
+        message.author.removeRole(role).catch(logger.error);
+        message.channel.send(`Yor are no longer a member of: ${args.join(' ') }... \nSorry to see you go` );

```

Comment:
Not for this update unless you want to but we should consider adding this to the generator.